features/support/worlds/benefit_market_world.rb

```diff
@@ -146,7 +146,7 @@ def generate_initial_catalog_products_for(coverage_kinds)
     health_products
     dental_products if coverage_kinds.include?(:dental)
     BenefitMarkets::Products::HealthProducts::HealthProduct.each do |hp|
-      qhp = create(:products_qhp, active_year: hp.active_year, standard_component_id: hp.hios_id[0..13])
+      qhp = create(:products_qhp, active_year: hp.active_year, standard_component_id: hp.hios_id[0..13)
```

Comment:
I guess there is a typo
`qhp = create(:products_qhp, active_year: hp.active_year, standard_component_id: hp.hios_id[0..13)`
->
`qhp = create(:products_qhp, active_year: hp.active_year, standard_component_id: hp.hios_id[0..13])`